.circleci/dist_compile.yml

```diff
@@ -391,7 +391,7 @@ jobs:
       - run:
           name: "Install Adapter Dependencies"
           command: |
-            mkdir ~/adapter-deps ~/adapter-deps/install
```

Comment:
Please make this change back in .config/circleci.yml. ( Apologies, for reverting this ).

Reply:
@kgpai I reverted it back to config.yml.